src/playwright-runner.ts

```diff
@@ -274,10 +274,18 @@ async function runPlaywright(
   fs.copyFileSync(path.join(__dirname, configFileName), configFile);
 
   const defaultArgs = {
-    output: runCfg.playwrightOutputFolder,
     config: configFile,
   };
 
+  // Set the Playwright output folder if SAUCE_SYNC_WEB_ASSETS is not enabled.
+  if (
+    !suite.env?.SAUCE_SYNC_WEB_ASSETS ||
+    suite.env.SAUCE_SYNC_WEB_ASSETS.toLowerCase() !== 'true'
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (defaultArgs as any).output = runCfg.playwrightOutputFolder;
+  }
```

Comment:
I think you're better off defining that [here](https://github.com/saucelabs/sauce-playwright-runner/blob/537c4d90130be88835f4a52c95553eded90f9b47/src/playwright-runner.ts#L177-L177). Less 🍝 that way.